lib/components/narrative/metro/default-route-renderer.tsx

```diff
@@ -25,21 +25,26 @@ const Block = styled.span<{ color: string; isOnColoredBackground?: boolean }>`
   `}
 `
 
-type RouteRendererProps = {
+export type RouteRendererProps = {
+  fullRender?: boolean
```

Comment:
Should we rename this to `expandedRender` or `renderName`?

Reply:
I dropped `fullRender` in https://github.com/opentripplanner/otp-react-redux/pull/904/commits/acc109a2bab19d7e623039169d06c0fc4ab30017. For immediate purposes the custom route renderer uses `leg.onColoredBackground`.

Reply:
I would be tempted to refactor that `leg.onColoredBackground` TBH.